cmd/receipts/main.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
+
+	"github.com/joho/godotenv"
+	kClient "github.com/kardiachain/go-kaiclient/kardia"
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>could not import github.com/kardiachain/go-kaiclient/kardia (-: could not load export data: no export data for "github.com/kardiachain/go-kaiclient/kardia") (typecheck)